src/components/TodoCount.js

```diff
@@ -0,0 +1,13 @@
+const TodoCount = ({$todoCount, currentState, currentFilterState}) => {
+    let todoCount = 0;
+    if(currentFilterState === "all") {
+        todoCount = currentState.length
+    } else if (currentFilterState === "active") {
+        todoCount = currentState.filter(todo => todo.isCompleted === false).length
```

Comment:
```suggestion
        todoCount = currentState.filter(todo => !todo.isCompleted).length
```

취향 차이일 수도 있으나, 보통 true, false 같은 boolean 타입 연산은 비교하기보다는
!형식을 더 많이 쓰는거 같아요.

Reply:
흠 더 깔끔해보이긴 하네요! 뭔가 한번 더 생각은 하게되지만, 통상적인 방향으로 짜보도록 하겠습니다 :)